message/query_test.go

```diff
@@ -226,7 +226,7 @@ func TestQueryCodec_Encode(t *testing.T) {
 				nil,
 			},
 			{
-				"missing query",
+				"query with empty query string",
 				&Query{},
 				nil,
 				errors.New("cannot write QUERY empty query string"),
```

Comment:
encode should probably be changed too so that the library can encode query messages with empty query strings, it's interesting that the Graph workload works without having to change the encode function here

Reply:
That's because the proxy is acting as a server wrt the client sending the QUERY message, and so it only needs to decode such messages, never encode.

Reply:
yeah I went back and checked, the proxy only calls `encode` when it modifies the request but in this case the Graph requests are not modified so the proxy never calls `encode` 👍